website/www/site/assets/scss/_navbar-desktop.scss

```diff
@@ -0,0 +1,196 @@
+/*!
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@import "media";
+
+.navigation-bar-mobile {
+  display: none;
+  .arrow-icon {
+    display: flex;
+    align-items: baseline;
+    margin-left: 10px;
+  }
+}
+.nav-tabs {
+  border-bottom: none;
+}
+.navigation-bar-desktop {
+  display: flex;
+  height: 96px;
+  width: 100%;
+  align-items: center;
+  margin-bottom: 30px;
+  box-shadow: 0 4px 16px 0 rgba(0, 0, 0, 0.06);
+  background-color: $color-white;
+  z-index: 10000; // just to make sure that navbar always on top of other elements
+
+  #iconsBar {
+    display: flex;
+  }
+
+  a {
+    @extend .component-text;
+    font-weight: 500;
+    color: $color-dark-gray;
+    letter-spacing: 0.2px;
+    line-height: 1.63;
+    margin-right: 39px;
+    text-decoration: none;
+    cursor: pointer;
+  }
+
+  .navbar-logo {
+    margin-left: 58px;
+    margin-bottom: 4px;
+    img {
+      width: 88px;
+    }
+  }
+  .navbar-bar-left {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+  .navbar-links {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 10000;
+    :last-child {
+      margin-right: 0;
+    }
+
+    .navbar-link {
+      display: inline-block;
+      position: relative;
+      margin-bottom: 2px;
+    }
+
+    .navbar-link::before {
+      transition: 0.3;
+      content: "";
+      position: absolute;
+      background-color: $color-sun;
+      height: 0%;
+      width: 100%;
+      bottom: 0px;
+      border-radius: 5px;
+    }
+
+    .navbar-link:hover::before {
+      height: 2px;
+    }
+
+    .navbar-dropdown-documentation {
+      list-style-type: none;
+
+      .dropdown-toggle {
+        margin: 0;
```

Comment:
@Jakub-Sadowski I'm pretty sure this is the cause of the navbar bug

It's overriding the `margin-right: 39px` in `.navigation-bar-desktop a`. When I turn it off in developer tools the navbar looks like I'd expect:
![image](https://user-images.githubusercontent.com/675055/106935300-0818b600-66d0-11eb-8778-437abf06c83d.png)

Can we try removing this before the final merge?

Reply:
Yes, I deleted it already

Reply:
Thanks, LG